### PR TITLE
update quay.io/openshiftdo/init image to 0.10.0

### PR DESCRIPTION
**Pull request by kadel**
Update init image to `quay.io/openshiftdo/init:0.10.0` 

The new version of the image includes custom inner loop scripts, see https://github.com/openshift/odo-init-image/pull/27

To test this just make sure that everything works as before :-)

### CODE REVIEW

**Comment by mik-dass** [comment]
```
2019/09/03 10:02:15 Container lease in pod benchmark completed successfully
2019/09/03 10:06:40 Container teardown in pod benchmark completed successfully
2019/09/03 10:06:40 Pod benchmark succeeded after 41m32s
2019/09/03 10:06:40 error: unable to signal to artifacts container to terminate in pod benchmark, triggering deletion: could not run remote command: container artifacts is not valid for pod benchmark
2019/09/03 10:06:40 error: unable to retrieve artifacts from pod benchmark: could not read gzipped artifacts: container artifacts is not valid for pod benchmark
2019/09/03 10:06:45 error: could not wait for pod 'benchmark': it is no longer present on the cluster (usually a result of a race or resource pressure. re-running the job should help)
2019/09/03 10:06:46 Ran for 45m11s
2019/09/03 10:06:47 Submitted failure event to sentry (id=86c290b95263455d8bc22c881b4e512e)
error: could not run steps: step benchmark failed: template pod "benchmark" failed: pod was deleted while ci-operator step was waiting for it
```
/retest

**Comment by kadel** [comment]
/retest

```
2019/09/03 10:59:25 error: unable to signal to artifacts container to terminate in pod integration, triggering deletion: could not run remote command: container artifacts is not valid for pod integration
2019/09/03 10:59:25 error: unable to retrieve artifacts from pod integration: could not read gzipped artifacts: container artifacts is not valid for pod integration
2019/09/03 10:59:30 error: could not wait for pod 'integration': it is no longer present on the cluster (usually a result of a race or resource pressure. re-running the job should help)
E0903 10:59:30.515219      15 event.go:191] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:".15c0e88ed6e9ab02", GenerateName:"", Namespace:"ci-op-kgd6z5k8", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Initializers:(*v1.Initializers)(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"", Namespace:"ci-op-kgd6z5k8", Name:"", UID:"", APIVersion:"", ResourceVersion:"", FieldPath:""}, Reason:"CiJobFailed", Message:"Running job pull-ci-openshift-odo-master-integration for PR https://github.com/openshift/odo/pull/2072 in namespace ci-op-kgd6z5k8 from author kadel", Source:v1.EventSource{Component:"ci-op-kgd6z5k8", Host:""}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xbf53afc49e9c7702, ext:5876098307858, loc:(*time.Location)(0x26765e0)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xbf53afc49e9c7702, ext:5876098307858, loc:(*time.Location)(0x26765e0)}}, Count:1, Type:"Warning", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events ".15c0e88ed6e9ab02" is forbidden: unable to create new content in namespace ci-op-kgd6z5k8 because it is being terminated' (will not retry!)
```

**Comment by kadel** [comment]
```
2019/09/04 16:25:56 Resolved source https://github.com/openshift/odo to master@a2b51999, merging: #2072 de0b5c0d @kadel
2019/09/04 16:25:56 Using namespace ci-op-01jqg3zq
2019/09/04 16:25:56 Running root, [release-inputs], src, [images], [release:latest], benchmark
2019/09/04 16:25:56 Creating namespace ci-op-01jqg3zq
2019/09/04 16:25:56 Creating rolebinding for user kadel in namespace ci-op-01jqg3zq
2019/09/04 16:25:57 Setting a soft TTL of 1h0m0s for the namespace
2019/09/04 16:25:57 Setting a hard TTL of 12h0m0s for the namespace
2019/09/04 16:25:57 Setting up pipeline imagestream for the test
2019/09/04 16:25:57 Created secret benchmark-cluster-profile
2019/09/04 16:25:57 Building root
2019/09/04 16:25:57 Tagged shared images from ocp/4.1:${component}, images will be pullable from registry.svc.ci.openshift.org/ci-op-01jqg3zq/stable:${component}
2019/09/04 16:26:01 Create release image registry.svc.ci.openshift.org/ci-op-01jqg3zq/release:latest
2019/09/04 16:26:23 Copied 1.14Mi of artifacts from release-latest to /logs/artifacts/release-latest
2019/09/04 16:27:50 Build root succeeded after 2m1s
2019/09/04 16:27:50 Building src
2019/09/04 16:29:25 Build src succeeded after 1m34s
2019/09/04 16:29:26 Executing template benchmark
2019/09/04 16:29:26 Creating or restarting template instance
2019/09/04 16:29:26 Waiting for template instance to be ready
2019/09/04 16:29:28 Running pod benchmark
2019/09/04 16:29:35 Container cli in pod benchmark completed successfully
Lease acquired, installing...
level=warning msg="Found override for ReleaseImage. Please be warned, this is not advised"
level=info msg="Consuming \"Install Config\" from target directory"
level=info msg="Creating infrastructure resources..."
level=error
level=error msg="Error: Error applying plan:"
level=error
level=error msg="1 error occurred:"
level=error msg="\t* module.vpc.aws_lb.api_external: 1 error occurred:"
level=error msg="\t* aws_lb.api_external: timeout while waiting for state to become 'active' (last state: 'provisioning', timeout: 10m0s)"
level=error
level=error
level=error
level=error
level=error
level=error msg="Terraform does not automatically rollback in the face of errors."
level=error msg="Instead, your Terraform state file has been partially updated with"
level=error msg="any resources that successfully completed. Please address the error"
level=error msg="above and apply again to incrementally change your infrastructure."
level=error
level=error
level=fatal msg="failed to fetch Cluster: failed to generate asset \"Cluster\": failed to create cluster: failed to apply using Terraform"
2019/09/04 16:41:27 Container setup in pod benchmark failed, exit code 1, reason Error
Another process exited
2019/09/04 16:41:42 Container test in pod benchmark failed, exit code 1, reason Error
2019/09/04 16:44:38 Copied 7.39Mi of artifacts from benchmark to /logs/artifacts/benchmark
2019/09/04 16:44:44 Ran for 18m47s
2019/09/04 16:44:44 Submitted failure event to sentry (id=ee88e0c4e6f24e09bf6b8cd27c044caf)
error: could not run steps: step benchmark failed: template pod "benchmark" failed: the pod ci-op-01jqg3zq/benchmark failed after 15m8s (failed containers: setup, test): ContainerFailed one or more containers exited
 Container setup exited with code 1, reason Error
---
Lease acquired, installing...
level=warning msg="Found override for ReleaseImage. Please be warned, this is not advised"
level=info msg="Consuming \"Install Config\" from target directory"
level=info msg="Creating infrastructure resources..."
level=error
level=error msg="Error: Error applying plan:"
level=error
level=error msg="1 error occurred:"
level=error msg="\t* module.vpc.aws_lb.api_external: 1 error occurred:"
level=error msg="\t* aws_lb.api_external: timeout while waiting for state to become 'active' (last state: 'provisioning', timeout: 10m0s)"
level=error
level=error
level=error
level=error
level=error
level=error msg="Terraform does not automatically rollback in the face of errors."
level=error msg="Instead, your Terraform state file has been partially updated with"
level=error msg="any resources that successfully completed. Please address the error"
level=error msg="above and apply again to incrementally change your infrastructure."
level=error
level=error
level=fatal msg="failed to fetch Cluster: failed to generate asset \"Cluster\": failed to create cluster: failed to apply using Terraform"
---
Container test exited with code 1, reason Error
---
Another process exited
---
```
/retest

**Comment by kadel** [comment]
/test all

retriggering, just to make sure that there are no flakes

File: tests/integration/cmd_push_test.go [hunk 1]
```diff
@@ -257,8 +262,6 @@ var _ = Describe("odo push command tests", func() {
 
 			Expect(modifiedCatViewFile).NotTo(Equal(earlierCatViewFile))
 			Expect(modifiedCatServerFile).To(Equal(earlierCatServerFile))
-
-			helper.HttpWaitFor("http://"+url, "UPDATED!", 30, 1)
```
Comment (amitkrout) [inline tests/integration/cmd_push_test.go]: Not an issue with moving it just after pushing the component for second time

/lgtm

**Pull request merged**